src/main/java/no/unit/nva/bare/Authority.java

```diff
@@ -1,18 +1,34 @@
 package no.unit.nva.bare;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.Collection;
 import java.util.HashSet;
 import java.util.Set;
 
 public class Authority {
 
     public static final String EMPTY_STRING = "";
+
+    @JsonProperty("name")
     private String name;
+
+    @JsonProperty("systemControlNumber")
     private String systemControlNumber;
+
+    @JsonProperty("feideids")
```

Comment:
Jeg hadde forventet feideIds

Reply:
I didn't want to change the existing object returned to frontend, but I will check what is actually returned as of now before merging my change.